src/c4/fc4/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+#-----------------------------*-cmake-*----------------------------------------#
+# file   c4/fc4/CMakeLists.txt
+# author Kelly Thompson <kgt@lanl.gov>
+# date   2020 April 20
+# brief  Generate build project files for Fortran.
+# note   Copyright (C) 2016-2020, Triad National Security, LLC.
+#        All rights reserved.
+#------------------------------------------------------------------------------#
+cmake_minimum_required(VERSION 3.17.0)
+project( fc4 C Fortran )
+
+#------------------------------------------------------------------------------#
+# If this directory is built via CAFS (ie. MSVC or XCode), then we need to do
+# some extra setup.  When build via CAFS, target Lib_dsxx won't be defined yet.
+#------------------------------------------------------------------------------#
+set(target_deps Lib_c4 Lib_dsxx )
+if(NOT TARGET Lib_dsxx)
+  include(CMakeAddFortranSubdirectory)
+  CAFS_Fortran_dir_boilerplate_setup()
+  CAFS_Fortran_dir_MPI_setup() # returns CAFS_MPI_DEPS
+elseif("${DRACO_C4}" STREQUAL "MPI")
+  list(APPEND target_deps MPI::MPI_Fortran)
+  add_definitions( -DC4_MPI )
+endif()
+
+# ---------------------------------------------------------------------------- #
+# Source files
+# ---------------------------------------------------------------------------- #
+file( GLOB f90sources *.f90 *.F90 )
+
+# ---------------------------------------------------------------------------- #
+# Build library
+# ---------------------------------------------------------------------------- #
+add_component_library(
+  TARGET       Lib_c4_fc4
+  LIBRARY_NAME fc4
+  SOURCES      "${f90sources}"
+  LINK_LANGUAGE Fortran
+  TARGET_DEPS   "${target_deps}")
+target_include_directories( Lib_c4_fc4 PUBLIC
+    $<INSTALL_INTERFACE:${DBSCFG_IMPORT_PREFIX}/fc4> )
+if( WIN32 )
+  # The library must be installed in the master project's RUNTIME_OUTPUT_DIR
+  set_target_properties( Lib_c4_fc4 PROPERTIES
+    RUNTIME_OUTPUT_DIRECTORY ${draco_BINARY_DIR}/${CMAKE_BUILD_TYPE} )
```

Comment:
Huh! I don't think I've ever used the RUNTIME_OUTPUT_DIRECTORY variable before. Good to know it exists!

Reply:
While it can be useful in general, it is important for Windows-based builds that require `.exe` and `.dll` files to be in the same directory.  I use this to avoid copy all those `.dll` files into the test directories.

Reply:
Ahh, interesting! Thanks :)